examples/annotation_label_studio/materializers/fastai_learner_materializer.py

```diff
@@ -0,0 +1,62 @@
+import os
+import tempfile
+from typing import Type
+
+from fastai.learner import Learner
+from fastai.vision.all import *
```

Comment:
`import *` probably a bad pattern, would encourage explicit imports

Reply:
This is the way you do it with fastai. They don't do explicit imports.

Reply:
Ok lets leave it in then although i am at a loss why they promote this

Reply:
I think you would not like many things they do. That's why we have to be careful about the integration when we do actually do it...